Source/WarpX.cpp

```diff
@@ -1395,6 +1395,7 @@ WarpX::AllocLevelMFs (int lev, const BoxArray& ba, const DistributionMapping& dm
     IntVect jx_nodal_flag, jy_nodal_flag, jz_nodal_flag;
     IntVect rho_nodal_flag;
     IntVect phi_nodal_flag;
+    amrex::IntVect F_flag, G_flag;
```

Comment:
Are the F and G flags also nodal flags or flag something else?

Reply:
I can call them `nodal`, too. They are the same type of flag, so yeah I'll call them the same way, so that it's crystal clear.